bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/FileInfo.java

```diff
@@ -80,6 +80,9 @@
     private int stateBits;
     private boolean needFlushHeader = false;
 
+    // lac
+    private Long lac = null;
```

Comment:
Is this used as an optimization to quickly find the highest LAC sent by the writer?